### DISCUSSION TITLE
Bitcoin Tokyo 2024 attendance

### DISCUSSION THREAD

**Discussion by junderw** [rust-bitcoin/rust-bitcoin#3042] [General]
September 21-22 there's a Bitcoin conference in Tokyo. I will be in attendance.

https://bitcoinconference.tokyo/en/

Anyone else coming? I would love to meet up.

**Comment by jirijakes** [comment]
I will be there, too.

**Comment by apoelstra** [comment]
Not me, sorry.

**Comment by TheBlueMatt** [comment]
I will be around the week after for a work offsite, not for the conference, but could possibly meet up.

**Comment by Kixunil** [comment]
I don't think we'll release `async` support until then...
